gatsby/content/projects/clients/mnotify.mdx

```diff
@@ -3,13 +3,13 @@ layout: projectimage
 title: mnotify
 categories:
  - client
-description: A simple matrix cli client. Primarily designed for sending notifications.
+description: A matrix cli client
 author: Stefan Tatschner
 language: Rust
 license: MIT
 repo: https://github.com/rumpelsepp/mnotify/
-# room: "#my-first-matrix-projects-room:homeserver.tld" # nb the quotes are needed as rooms start with #
-# thumbnail: /docs/projects/images/upload-a-thumbnail-image-to-the-images-subfolder.png
+room: https://matrix.to/#/#mnotify:hackbrettl.de
```

Comment:
```suggestion
room: #mnotify:hackbrettl.de
```

Can you keep the room alias only to be consistent with the rest of the projects :) ?

Reply:
oh yes, my bad!